### PR TITLE
Add ERD and database with test data

### PR DESCRIPTION
**Pull request by yash-puligundla**
The following changes have been added as a part of this PR
- ERD
- database creation and population scripts for postgresql and sqlite3
- Makefile with database creation, population commands for postgres and sqlite3

### CODE REVIEW

**Review by jb-adams — COMMENTED** [review]
Looks great!

Here are some of my ideas:

- Can we add a `platform` table upstream of `report_series`, a platform means a web service or series of services (e.g. ENA's refget service). We can connect platform to report series via foreign key, essentially saying "this report series is for this platform." One platform may have many report series, but each report series belongs to only one platform (one-to-many)

- Can we add an `organization` table upstream of `platform`. Organization being the implementing or hosting org (e.g. ENA, ELIXIR, DNAstack, GA4GH). One organization can have many platforms

- Can we add a `testbed` table upstream of `report_series`. Testbed means the actual application that is a valid GA4GH testbed (e.g. refget compliance, RNAget compliance, DRS + Passport). A `testbed_id` in `report_series` foreign key indicates what testbed the  series is associated with. 

- Delete `testbed_name`, `testbed_version`, and `testbed_description` from `report`, since that info can be gained from data joins to `testbed`

- Delete `platform_name` and `platform_description` from `report_series`, as that can be gained from data joins to `platform`

**Comment by yash-puligundla** [comment]
- Updated PostgreSQL scripts to reflect the new ERD changes.
- changed the table name from "user" to "github_user" as user is a reserved keyword in PostgreSQL.

File: database/postgresql/create-tables.sql [hunk 1]
```diff
@@ -0,0 +1,190 @@
+CREATE TABLE IF NOT EXISTS ga4gh_testbed_summary
```
Comment (jb-adams) [inline database/postgresql/create-tables.sql:1]: rename table to `summary`

File: database/ERD/ga4gh-testbed-erd.drawio [hunk 2]
```diff
@@ -0,0 +1 @@
...ybWRIezRQO7S1UYtrh6u3CSBD2llhUJXPCueoHMiQtjfOHyfH23qQhT5FUAJQVWDNSB1UxdWbBpDhbTaU5iYBrBpZzQV2kW6DD4BKAgtI6qAkrt81gAwNriFJBFp7G68jiioN+ghaLf1jy5gGLNB2x+NzJfaW7p32F/EGKir0FrloD5TvLXL5oaFN3QzX7H7SFdBRgxP3ZEdpeN4iAVzAi1R10QQo/3Qnrh2uXzfDNdsE6KIJsJoLpgkQvUVCKAGoKtAQqIOquH4NDZe2AxqlKtASWM5FEZx8rCrKVtNAtQQ2k2lADJZp1UFLaFBDo0jc08eLIFFJCLT9Nl9JCFRFMID+BUQPkP4FM5/bpdat90l1rXPn8hy44rVQ+qhgvh2NMvnpo0PtWgi4xUBFHpVzcfrosD7UsD6U6gy6Uafpo5IxflGKdGcYL4tcc4Dc1g3kDpXYfHG6/7A+lN11jvRIMN8fQa4C5Po1gxnUW7hc3AumXx+JdN0MZsRRFVhKTQtQ+xT+gi2FIUUfdDu1gi13ddrlRvegPlLnVS5GtL3+aevHFMIhxmkRuxaoZbuMMxoTauoitUaY1y8/UmskJa+fhQR1lrPCxWNqpFYJdDSekTFtL88L0IJtdCQAj9MGtjKIXQ5cuIbrBC8CVnXQSkLEvyKuJFgBWx0rCe2s7hLdKyXKUUOQMYdJXTNt0JHoAbI/huvqwP6sGKyO2V+75uwy2R+Ddau5EOhcZgL7y4jANZb9MQJXB/ZnxlZ1TP8mp+KPMQC3jPkoYKWPoNXSv4zQWkPp37ZoU2Hm0JlE8cILg5/evqdhkz7owfDu1LPwmc6dbtPwbQvtdtKdO9WKuJ00fNsyOw2/Ajpqb9vCNPxLcAEubNq20K4n/WR3gXa4ehq+bRls16tQjqrBtmi7XhztHjZ3+aev7tL//PBpDSa1UgQhAJUEmv90UBJXT8C3LaMT8Cuco5oou4UdJ+DvzD7sMDEoigJz8BtAgzn4OiiK6+fgp49gtKYg/DIxX1PQdsVUUxz6BoCqCgGQAFQVop3Ku2zr3HECaPlGeQJoEeF+NkWu3OfokyI3rjXCdeo+M+4UuVFtpEF9JMUZcjYRTOTXHfwtW0BbAovmcvBzt4AuiVsf7BOrXwP/xdnP49pIA2nJz+nbOIqSw9tjb7P8HM387I7/Bw==</diagram></mxfile>
```
Comment (jb-adams) [inline database/ERD/ga4gh-testbed-erd.drawio:1]: Can you rename table `user` to `github_user` in the ERD?

**Review by jb-adams — CHANGES_REQUESTED** [review]
Looks great @yash-puligundla ! I think it's just about there, just a couple edits

**Comment by yash-puligundla** [comment]
@jb-adams  Good catch. I have removed the duplicate summary table and changed "user" to "github_user" on the ERD

**Review by jb-adams — APPROVED** [review]
LGTM!

**Pull request merged**